mapbox/app/build.gradle

```diff
@@ -59,6 +59,11 @@ dependencies {
     // Google Play Services
     compile rootProject.ext.dep.gmsLocation
 
+    // LOST
+    compile(rootProject.ext.dep.lost) {
+        exclude group: 'com.google.guava'
```

Comment:
If we upgrade the dependency to the latest LOST that Guava exclusion is no longer necessary.

Reply:
I tested the sample app with lost `3.0.3` (atm latest version) and I found that it wasn't working properly so I'd rather keep lost `1.1.1` until https://github.com/mapzen/lost/issues/241 gets fixed.

Reply:
@Guardiola31337 I'd actually argue the opposite. With this PR the developer now has the freedom to switch location providers if they don't meet their needs. I think that all that https://github.com/mapzen/lost/issues/241 shows is that LOST fails under certain scenarios. I'd still upgrade to the latest version of LOST and leave it up to the developer decide if that scenario is a blocker for them.